exercises/practice/word-count/.meta/config.json

```diff
@@ -1,6 +1,17 @@
 {
   "blurb": "Given a phrase, count the occurrences of each word in that phrase.",
   "authors": [],
```

Comment:
author not on github

https://github.com/exercism/csharp/tree/4d73a89429b3eda350c3107c4f36091e792284a6/word-count